addons/toolbars/src/components/MenuToolbar.tsx

```diff
@@ -43,6 +43,7 @@ export const MenuToolbar: FC<MenuToolbarProps> = ({
       {selectedIcon ? (
         <IconButton key={name} active={active} title={description}>
           <Icons icon={selectedIcon} />
+          {showName ? `\xa0${name}` : ``}
```

Comment:
what's `\xa0` and I think we should return `null` rather than an empty string.

Reply:
agree about `null`

Reply:
`\xa0` is for the space character, because when I simply leave a space the icon and the label are not separated